wp-content/themes/cpnet-developer/single.php

```diff
@@ -19,8 +19,8 @@
 
 			<?php //wporg_developer_post_nav(); ?>
 
-		<?php endwhile; // end of the loop. ?>
+		<?php endwhile;?>
 
-		</main><!-- #main -->
+		</main><?php get_sidebar(); ?><!-- #main -->
```

Comment:
This is where I've somehow added code to display a widgets area.

Reply:
Thanks. Yes, it doesn't have sidebars right now. I will add it.